scripts/galvo_scan_to_multiprobe.py

```diff
@@ -150,47 +132,41 @@ def get_galvo_rotation(ccd, channel=None, galvo_offset=3):
     ----------
     ccd: (odemis.model.DigitalCamera)
         A camera object of the diagnostic camera.
-    channel: (int)
-        Channel number of waveform generator. If None alignment should be done manually. Default None.
-    galvo_offset: (float)
+    galvo_y_offset: (float)
+        Experimentally determined offset of the DeScan-X galvo in Volt.
+    galvo_x_offset: (float)
         Experimentally determined offset of the DeScan-Y galvo in Volt.
 
     Returns
     -------
     galvo_rotation: (float)
         The clockwise angle, in radians, of the DeScan-Y galvo relative to the y-axis of the diagnostic camera.
+    scale: (tuple of floats)
+        The x and y scale factor between the voltages and coordinates, in pixels per volt.
```

Comment:
Check if gain and scale are depicting the same, that only one term is used. Otherwise it gets confusing.